### DISCUSSION TITLE
AssertionError: Inconsistent number of segment

### DISCUSSION THREAD

**Discussion by 123ruler** [linto-ai/whisper-timestamped#52] [Q&A] [answered]
Hi.

Recently, when using whisper_timestamped, I often got an error message like this: 

/usr/local/lib/python3.8/dist-packages/whisper_timestamped/transcribe.py in _transcribe_timestamped_efficient(model, audio, remove_punctuation_from_words, compute_word_confidence, include_punctuation_in_confidence, refine_whisper_precision_nframes, alignment_heads, plot_word_alignment, word_alignement_most_top_layers, detect_disfluencies, trust_whisper_timestamps, use_timestamps_for_alignment, **whisper_options)
    799     if l1 != l2 and l1 != 0:
    800         logger.warning(f"Inconsistent number of segments: whisper_segments ({l1}) != timestamped_word_segments ({l2})")
--> 801     assert l1 == l2 or l1 == 0, f"Inconsistent number of segments: whisper_segments ({l1}) != timestamped_word_segments ({l2})"
    802 
    803     logger.debug("Compile results")

AssertionError: Inconsistent number of segments: whisper_segments (491) != timestamped_word_segments (490)

The only way I can solve this problem right now is to change Whisper's model from medium to base, and it will work properly.
Is there a good way to solve this problem? I would appreciate any helpful answers to this question. Thanks.

**Comment by Jeronymous** [comment, marked as answer]
This issue is described in https://github.com/linto-ai/whisper-timestamped/issues/47

It should be fixed now (update whisper-timestamped).

And I recommend also to use for now
```
openai-whisper==20230124
```
(because that version of whisper is super recent and might still include bugs. see https://github.com/openai/whisper/discussions/1058 and I see that whisper repo is moving a lot now)

**Reply by 123ruler** [reply]
Thank you very much! Now everything is back to normal!
